server/views/pages/confirmDISFormsIssued.njk

```diff
@@ -61,17 +61,17 @@
         },
         { text: 'Name and prison number',
           attributes: {
-          "aria-sort": "none"
-        }
+            "aria-sort": "none"
+          }
         },
         { text: "Discovery date and time",
           classes: 'confirmDISFormTable-discDateTime',
           attributes: {
-          "aria-sort": "descending"
-        }
+            "aria-sort": "ascending"
```

Comment:
I checked that this was displaying correctly previously, I guess the tables weren't actually working correctly before?